Feature/list

jadie0319 opened the pull request.

Commit:
08/10/18

 Commit Test

Commit:
BoardDAO 에 getBoardList() 생성, DBUtil 생성. /board/list 페이지 출력.

siyoon210 commented:
거의 다 완성된거같네요 
JSP부분 남으신거죠

siyoon210 commented on src/main/java/com/fastcampus/jcjboard/dao/BoardDao.java:
```diff
@@ -1,5 +1,54 @@
+        try {
+
+            conn = DbUtil.connect(dbUrl,dbId,dbPassword);
+            String sql ="select boardid,nickname,title,content from board";
```
쿼리문에 와일드카드로 가져오는게 더 나을거같아요. 날짜나, 글번호도 출력하도록요
->SELECT * FROM board

jadie0319 replied:
아 글번호는 boardid 라 이미 되어있고 날짜는 지금 추가했는데 이게 문제가 좀 생겨서 내일 얘기를 좀 해봐야 할듯...ㅠ

---- 메모. 내일 상의해야할 부분.
1.날짜에서 문제발생. BoardDO 에서 LocalDateTime변경?? 
2.DO --> VO 로 바꾸기. (그냥 단순한 파일명 문제이지만 파일 하나만 바꾸면 되는거니 바꿔도 될듯)
3.근데 LocalDateTime 변경해도 날짜는 나오는데 시간이 이상하게 나옴.

jsp 에 날짜빼고 디비에 있는 리스트들 출력은 성공.

jadie0319 replied:
![image](https://user-images.githubusercontent.com/10750614/46676982-9bcf3a00-cc1c-11e8-95f8-c32096ea5853.png)

siyoon210 replied:
저도 전에 게시판 혼자할떄 DB에서 시간을 가져오면 LocalDateTime으로 받아도 시간이 안나오더라구요 일단은 날짜만 표시하는걸로 구현하고 추후에 수정해봐요.

jadie0319 replied:
오키. 아 그리고 내꺼 DAO  BoardDaoList 로 바꿈.
지금 이 풀 리퀘스트는 머지만 하지 않으면 상관없으니. 
내일 여기 적힌 내용들 상의한 후에 냅두거나 지우기로 하고. 내꺼 추가된 내용은 내일 풀리퀘스트 할게.

Commit:
08/10/18

 날짜에서 문제발생. BoardDO 에서 LocalDateTime변경
 DO --> VO 로 바꾸기.
 list.jsp 게시판 리스트 출력부분 생성.

Commit:
10/10/18

 BoardListServlet  , list.jsp

siyoon210 committed:
Merge branch 'develop' into feature/list
